app/src/backend/ResultsService.ts

```diff
@@ -66,9 +68,10 @@ export async function getEmissionResults(inventoryId: string) {
     ]);
   const topSubSectorEmissionsWithPercentage = topSubSectorEmissions.map(
     // @ts-ignore
-    ({ co2eq, sector_name, subsector_name }) => ({
+    ({ co2eq, sector_name, subsector_name, scope_name }) => ({
```

Comment:
Can you also get rid of the ts-ignore by adding proper type definitions? (optional)